addons/purchase_stock/tests/test_stockvaluation.py

```diff
@@ -304,7 +305,6 @@ def setUpClass(cls):
         cls.supplier_location = cls.env.ref('stock.stock_location_suppliers')
         cls.stock_location = cls.env.ref('stock.stock_location_stock')
         cls.partner_id = cls.env['res.partner'].create({'name': 'Wood Corner Partner'})
-        cls.product1 = cls.env['product.product'].create({'name': 'Large Desk'})
```

Comment:
I dunno how this removal is related to the task here :thinking:

Reply:
Not it's not related just little clean up as they are re-assigning values before using it anywhere makes this line useless :see_no_evil: